lib/services/consumed/status-bar/status-bar.js

```diff
@@ -31,8 +30,8 @@ export class StatusBarConsumer {
 
     this.handleKernelCommand = handleKernelCommand;
```

Comment:
By removing line 17, `handleKernelCommand: Function;`, `handleKernelCommand` is now not defined as a class member before you reference it. If you do not need to save the function, remove this line and refactor the code elsewhere accordingly, or add line 17 back.

Reply:
hi @wadethestealth thank you very much for your thorough instruction and patience. I don't have npm in my computer yet. Let's see, if there are more issues, I will install npm and do flow typechecking.